oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/TokenHandlerTest.java

```diff
@@ -155,7 +155,6 @@ public class TokenHandlerTest {
 
     @BeforeEach
     void setUp() {
-        when(configurationService.getOidcApiBaseURL()).thenReturn(Optional.of(BASE_URI));
```

Comment:
How come this isn't needed any more? Was it never used?

Reply:
Yeah, not needed.